aries/agents/mediator/README.md

```diff
@@ -119,12 +119,7 @@ cargo build --package mediator --no-default-features --release
 Currently exposed endpoints.
 
 ```yaml
-`/register`:
-- **Description** : | 
-    Shows an Aries Out Of Band (OOB) invitation which can be used to connect to the mediator using a conformant Aries Agent.
-    Use `Accept` header with value "application/json" to receive same in json format.
-
-`/register.json`:
+`/invitation`:
 - **Description** : Returns OOB invitation in json format.
```

Comment:
Let's keep the expanded version of the `OOB` abbreviation in this description, like previously in `/register`. 
And a very brief purpose description. This would help orient newcomers, letting them understand what it's for at a glance.